src/docfx/build/page/BuildPage.cs

```diff
@@ -25,55 +25,42 @@ public static List<Error> Build(Context context, Document file)
             errors.AddRange(monikerErrors);
 
             var outputPath = context.DocumentProvider.GetOutputPath(file.FilePath);
-
-            var publishItem = new PublishItem(
-                file.SiteUrl,
-                outputPath,
-                context.SourceMap.GetOriginalFilePath(file.FilePath) ?? file.FilePath.Path,
-                context.BuildOptions.Locale,
-                monikers,
-                context.MonikerProvider.GetConfigMonikerRange(file.FilePath),
-                file.ContentType,
-                file.Mime);
-
             if (errors.Any(e => e.Level == ErrorLevel.Error))
             {
-                context.PublishModelBuilder.Add(file.FilePath, publishItem);
-                return errors;
+                context.ErrorLog.Write(errors);
+                context.PublishModelBuilder.Add(file.FilePath, null, null, outputPath);
```

Comment:
remove now that `PublishUrlMap` is the source of truth?
```suggestion
                ~~context.PublishModelBuilder.Add(file.FilePath, null, null, outputPath);~~
```